css/partials/_page-titles.scss

```diff
@@ -21,7 +10,7 @@
 		background: none;
 		color: #111;
 		display: block;
-		padding: 1rem;
+		padding: 0 5%;
 	}
 	a.title-page {
```

Comment:
@frrrances do you want me to move the new class selector (.title-page-link) outside the nest of .title-page?